src/clr-angular/layout/tabs/tabs.spec.ts

```diff
@@ -215,6 +216,20 @@ describe('Tabs', () => {
       context.fixture.detectChanges();
       expect(compiled.querySelector('.tabs-overflow .nav-item').getAttribute('role')).toEqual('presentation');
     });
+
+    it('does not reuse views with *clrIfActive', () => {
+      const tabsService = context.getClarityProvider(TabsService);
+      const ifActiveService = context.getClarityProvider(IfActiveService);
+      const originalId = ifActiveService.current;
+      const initialView = tabsService.tabContentViewContainer.get(0);
+      // leave current tab
+      ifActiveService.current = -1;
+      context.detectChanges();
+      // get back to initial tab
+      ifActiveService.current = originalId;
+      context.detectChanges();
+      expect(tabsService.tabContentViewContainer.indexOf(initialView)).toBe(-1);
```

Comment:
Nit: This is testing only on a service level. But we could make it stricter by testing directly on the projected content (whether the content elements are projected only once or not 🤷‍♂ ).